### DISCUSSION TITLE
Disable caching in PROD build

### DISCUSSION THREAD

**Discussion by mwskwong** [gatsbyjs/gatsby#33100] [Help]
Basically, I'm deploying my webpage to a service that doesn't support caching, so it is meaningless to spend a significant amount of time building cache.

Is there a way to disable caching when running `gatsby build`? Not just deleting it afterward, but not building it at all in the first place.

**Comment by herecydev** [comment]
There's `GATSBY_DISABLE_CACHE_PERSISTENCE` that I believe will do what you want, set it as an env variable to "1" or "true" or similar and see if that works for you.

**Reply by mwskwong** [reply]
Nope, does not appear to be working.

**Comment by mwskwong** [comment]
Bump.
